pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -2397,6 +2404,12 @@ const (
 	LogTruncateCommand LogSubCommandType = "log-truncate"
 	// LogStopCommand is the stop command of log backup.
 	LogStopCommand LogSubCommandType = "log-stop"
+	// LogPauseCommand is the pause command of log backup.
+	LogPauseCommand LogSubCommandType = "log-pause"
+	// LogResumeCommand is the resume command of log backup.
+	LogResumeCommand LogSubCommandType = "log-resume"
+	// LogUnknownCommand is the unknown command of log backup.
+	LogUnknownCommand LogSubCommandType = "log-unknown"
```

Comment:
is `log-unknown` set by users? or just an invalid value.

Reply:
The latter one, just in case that the user used some unexpected input, maybe `+kubebuilder:validation:Enum` is a better choice